izanami-frontend/src/pages/tenantSettings.tsx

```diff
@@ -90,6 +90,10 @@ export function TenantSettings(props: { tenant: string }) {
       </>
     );
   } else if (tenantQuery.data) {
+    const { data: usersData, isLoading } = useQuery(
```

Comment:
Hooks must never be called in conditional statements, [read this](https://react.dev/warnings/invalid-hook-call-warning#breaking-rules-of-hooks) for details.

I don't understand why we need to wait for tenantQuery here, since all we need for this query is tenant name, and we already have this from props. I think both query should be run in parallel.